electron/ironfish/IronFishManager.ts

```diff
@@ -269,6 +269,11 @@ export class IronFishManager implements IIronfishManager {
     if (peers.length > 0) {
       totalSequences = peers[0].sequence
     }
+
+    const client = await this.sdk.connectRpc()
```

Comment:
I would propose to connect to RPC after node [initialization](https://github.com/iron-fish/node-app/pull/250/files#diff-c598a6798f58a9d3f4589fce8df97ae89e5b351c8c10943acf7c8d8a05131f77L249) and store it in the class property.

Reply:
Also I suggest to add @skylarrichard123 to Reviewers, because to my mind all UI changes must be approved by designer.

Reply:
I tried making this change, but there is a complication due to the fact that the node needs to be started before you can connect to the RPC and there are places where the node has methods called before the RPC can be assigned.

I am going to leave it like this until we refactor and get rid of the reference to the IronfishNode and use only the SDK/RPC.